tests/test_solver.py

```diff
@@ -68,4 +68,4 @@ def test_solver_by_inversion(self):
         induced_moments_direct = np.linalg.inv(bmat) @ static_fields
         induced_moments_solver = cppe_state.get_induced_moments()
         np.testing.assert_almost_equal(induced_moments_solver,
-                                       induced_moments_direct, decimal=10)
+                                       induced_moments_direct, decimal=9)
```

Comment:
This is okay I guess. The polarizabilities in the potential file only have 8 digits, so probably asking for that tough convergence was a bit harsh anyways :smiley: